src/controller/data_model/controller-clusters.matter

```diff
@@ -2968,6 +2968,72 @@ provisional client cluster Timer = 71 {
   command ReduceTime(ReduceTimeRequest): DefaultSuccess = 3;
 }
 
+/** This cluster supports remotely monitoring and, where supported, changing the operational state of an Oven. */
+provisional client cluster OvenCavityOperationalState = 72 {
+  revision 1;
+
+  enum ErrorStateEnum : enum8 {
+    kNoError = 0;
+    kUnableToStartOrResume = 1;
+    kUnableToCompleteOperation = 2;
+    kCommandInvalidInState = 3;
+  }
+
+  enum OperationalStateEnum : enum8 {
```

Comment:
Would it make sense to define a separate enum for this as the values in the spec are slightly different? Specifically I see: Runnint, Preheating, Preheated, CoolingDown and SelfCleaning.

Reply:
The spec is being changed in flight on this PR. https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/8583/files. We are no longer having our own Op States for Oven but instead are utilizing specific verbiage for Phase List

Reply:
This file is no longer part of this PR. This change has already been merged to the master.